# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: thousand separator for Swiss locales is '
# DP: Related bugs: #345481
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: Denis Barbier
# DP: Upstream status: Fixed in CVS for de_CH, but not for it_CH
# DP: Date: 2006-08-12

Index: glibc-2.3.6/localedata/locales/it_CH
===================================================================
--- glibc-2.3.6.orig/localedata/locales/it_CH
+++ glibc-2.3.6/localedata/locales/it_CH
@@ -57,9 +57,7 @@
 END LC_MONETARY
 
 LC_NUMERIC
-decimal_point             "<U002C>"
-thousands_sep             "<U0027>"
-grouping                  3;3
+copy  "de_CH"
 END LC_NUMERIC
 
 LC_TIME
